crowd/actions/add_user_to_group.yaml

```diff
@@ -0,0 +1,17 @@
+---
+name: add_user_to_group
+runner_type: python-script
+description: "Add a user to a group."
+enabled: true
+entry_point: "add_user_to_group.py"
+parameters:
+  username:
+    type: "string"
+    description: "The username to assign to the group."
+    required: true
+    secret: true
```

Comment:
Username probably doesn't need to be a secret? Or is there a reason it's marked as a secret?